### ISSUE TITLE
Tool tip rendering double tooltip

### ISSUE THREAD

**Issue by TechStack** [shadowmage45/AncientWarfare#39]
as you can see in the attached screenshot tooltips are doubled  in some of the custom inventory screens.
![2013-12-28_22 19 12](https://f.cloud.github.com/assets/4016300/1818967/8ee198a0-7047-11e3-9088-46c549d9069a.png)

I'd call this a lower priority bug. Just wanted to let you know it existed

**Comment by shadowmage45** [comment]
Thanks for the report -- I will take a look at this and try to have it squashed for the next update

**Comment by shadowmage45** [comment]
took a look at the dev-code, and it appears that this has been fixed -- probably as part of the other GUI cleanup I was doing.  I will leave this open for now as I cannot currently duplicate it -- please let me know if it is still present in the next upcoming release.

**Comment by shadowmage45** [comment]
Still cannot duplicate -- closing for now.

Please re-open if you find it pops up again, and I'll take another/deeper look at it.

**Comment by TechStack** [comment]
This happened again today so I did some troubleshooting and was able to track it down to having NEI installed.
When NEI is loaded the tool tips get doubled
When it is Disabled ( not loaded) they do not.
I also tried just disabling the NEI -AW plugin and it still doubled the tool tips.

I think this issue should remain closed just wanted to provide feedback in case anyone else has this issue in the future.
